### PR TITLE
feat: display help when arriving

### PR DESCRIPTION
**Pull request by alexfauquette**
This code display a modal when opening the page for the first time or when clicking on the help button (top right). 

image

image

### CODE REVIEW

**Comment by alexfauquette** [comment]
As ou can see, I did not think about what to put inside, if you have instructions for games, you're welcome :)

**Comment by teolemon** [comment]
Just a thought: one welcome for all games, or one welcome per game ?
"The rules of the game"

**Comment by alexfauquette** [comment]
I propose : 

- On presentation displayed at welcome with : 
  - Highlight the button help
  - Present each game in one sentence
- A more detailed presentation per game with : 
  - rules and advice for the game
  - displayed only when the user clicks on the help button
  
Why: There are many games and everyone is not interested in every game. Moreover, some of them can be complex like the logo one, so rules must be too long for a new user.

File: src/components/Helper.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <img wrapped size="large" src="~/../assets/logo.png" />
+        <sui-modal-description>
+          <sui-header>Le logo</sui-header>
+          <p>
+            Ceci est un très joli logo
+          </p>
+          <p>Vous ne trouvez pas qu'il ressemble fort à une loupe ?</p>
+          <p>
+            Il faudrait ici metre un text et une image pour le jeu :
+            {{ $router.history.current.path }}
```
Comment (teolemon) [inline src/components/Helper.vue]:
```suggestion
          <sui-header>Welcome to Hunger Games</sui-header>
          <p>
            Hunger Games is a collection of mini-games that help contribute to Open Food Facts in many ways. Categorize a product to get the Nutri-Score and Eco-Score, help validate labels for the Eco-Score, add brands and weights, help our logo recognition system learn about the logos it detected.
          </p>
          <p>Those are some of the exciting things you'll be able to do. Make sure to be connected to the Open Food Facts website to get credit for your contributions. If you make a mistake, please take the time to correct it by heading to the website.</p>
          <p>
            {{ $router.history.current.path }}
```

**Comment by VaiTon** [comment]
Good job! We should use translatable text thought.

**Comment by alexfauquette** [comment]
[The netlify preview](https://deploy-preview-176--gifted-lalande-686eef.netlify.app/questions?type=label)

To be editable by anyone, explanations are in an object like this one : 

```javascript
{
logos: 
    [{
      title: "page1 title",
      texts: [
        "paragraph1",
        "second paragraph",
      ],
      image: "logoGeneral.png",
      imageSize: "huge",
    },
    {
      title: "second page title",
      texts: ["a paragraph" ],
    },
   ...
   ],
welcome : [...]
}
```

images must be added in the folder `assets`

- `logos` help appears only on the page of the logo game. If there is no explanation for a page, the welcome help is displayed
- if no image is specified, the OFF logo is used
-  if no image size is specified, `medium` is used

Things to fix : 
- [x] I still use raw text (I will replace all that with translation keys)
- [x] Images blink between explanation pages
- [x] The UI encourages the user quite the introduction before the end

File: src/components/Helper.vue [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            class="explanationText"
+          >
+            {{ text }}
+          </p>
+        </sui-modal-description>
+      </sui-modal-content>
+      <sui-segment aligned="right">
+        <sui-progress attached top :percent="pageRatio" color="green" />
+        <sui-button @click.native="prev">prev</sui-button>
+        <sui-button
```
Comment (hangy) [inline src/components/Helper.vue:25]: Would it make sense for this button to be blue (class `primary` on other views)? I think the 'next'  button does not have a strong call to action as-is:
image

**Review by hangy — COMMENTED** [review]
Could you move the strings to `/src/i18n`, so that they can be translated via Crowdin?

File: src/components/Helper.vue (hunk 2 above)
Comment (alexfauquette) [reply src/components/Helper.vue:25]: Good idea

**Comment by alexfauquette** [comment]
Texts are now in `src/i18n/en.json`

the only remaining part is the image name and there sizes

**Pull request merged**